modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsSnapshot.java

```diff
@@ -27,7 +27,9 @@
 
 /**
  * Metrics snapshot.
+ * @deprecated Replaced by CacheMetricsSnapshotV2 with versioning support.
  */
+@Deprecated
 public class CacheMetricsSnapshot implements CacheMetrics, Externalizable {
```

Comment:
Can we also rollback entryProcessor* externationalization from this class so that https://issues.apache.org/jira/browse/IGNITE-10925 will be fixed even for  CacheMetricsSnapshot (v1) users?